scripts/CompareFitMinimizers/fitting_benchmarking.py

```diff
@@ -336,38 +346,36 @@ def run_fit(wks, prob, function, minimizer='Levenberg-Marquardt', cost_function=
     @returns the fitted parameter values and error estimates for these
     """
     status = None
-    chi2 = None
     param_tbl = None
-    fit_wks = None
     try:
         # When using 'Least squares' (weighted by errors), ignore nans and zero errors, but don't
         # ignore them when using 'Unweighted least squares' as that would ignore all values!
         ignore_invalid = cost_function == 'Least squares'
-
         # Note the ugly adhoc exception. We need to reconsider these WISH problems:
         if 'WISH17701' in prob.name:
```

Comment:
This exception deserve a separate pull request to get fixed....

Reply:
sorry, which exception? The WISH one?

Reply:
Yes just WISH17701:)